manifests/o/Open-Shell/Open-Shell-Menu/4.4.190/Open-Shell.Open-Shell-Menu.locale.en-US.yaml

```
@@ -0,0 +1,26 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Open-Shell.Open-Shell-Menu
+PackageVersion: 4.4.190
+PackageLocale: en-US
+Publisher: Open-Shell

```

```suggestion
Publisher: The Open-Shell Team
```
This must match Control Panel.